index.php

```diff
@@ -41,9 +41,8 @@ function page_snippetslist_tab($page)
 
 function grouped_snippets()
 {
-    $snippets =Record::find(array(
+    $snippets =Snippet::find(array(
```

Comment:
I can't remember but I probally did so because I liked to support 0.7.3.
But now it makes no more sence to do so, since some security things 
where fixed.
Probally best to set require_wolf_version to 0.8.3

Op 24-7-2016 om 17:49 schreef Yuli C.:
>
> In index.php 
> <https://github.com/svanlaere/easysnippet/pull/2#discussion_r71990853>:
>
> > @@ -41,9 +41,8 @@ function page_snippetslist_tab($page)
> >
> >  function grouped_snippets()
> >  {
> > -    $snippets =Record::find(array(
> > +    $snippets =Snippet::find(array(
>
> Did the API change for this in 0.8.0 ?
> If so, version bump should be done, for Easysnippet and required Wolf 
> version.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub 
> <https://github.com/svanlaere/easysnippet/pull/2/files/21dc23dcbdc14f3a48dc9a2fd4bb8b7914beb8f9#r71990853>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AIJhfp05b670v7UwC3PiC7np9VxpU3BCks5qY4lxgaJpZM4JTdue>.
>